Assets/GRPC.NET/Scripts/GRPCBestHttpHandler.cs

```diff
@@ -184,11 +219,6 @@ protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage grpcRe
                         incomingDataStream.CloseWithException(ex);
                     }
                 }
-                // but in any case we want to close the stream to indicate to grpc that we are done
-                else
-                {
-                    incomingDataStream.Close();
```

Comment:
we can't close the data stream as part of concluding the request, as the download stream may still be draining in the background. so the download process is responsible for closing the stream on the happy path now. any exception paths can still close the stream with an exception though

Reply:
except in the case of a trailers only response where we get no `DATA`, that will still close the stream